scripts/lib/setup-yum-repo

```diff
@@ -38,26 +38,27 @@ fi
 yum update -y
 
 # "Development Tools" is the equivalent of build-essential
-yum groupinstall -y "Development Tools"
+yum groups mark install "Development Tools"
+yum update -y
 
 RHVER="$(rpm -qf --queryformat="%{VERSION}" /etc/redhat-release)"
 RHARCH="$(rpm -qf --queryformat="%{ARCH}" /etc/redhat-release)"
 PGVER=10
 if [ "$is_centos" = true ]; then
     # PostgreSQL $PGVER
-    yum localinstall -y "https://yum.postgresql.org/$PGVER/redhat/rhel-$RHVER-$RHARCH/pgdg-centos$PGVER-$PGVER-2.noarch.rpm"
+    yum reinstall -y "https://yum.postgresql.org/$PGVER/redhat/rhel-$RHVER-$RHARCH/pgdg-centos$PGVER-$PGVER-2.noarch.rpm"
```

Comment:
Can you point to docs indicating that `reinstall` will do the same thing as `localinstall` if the thing isn't already installed?

I'm not a yum expert so not sure if that's true off the top of my head.